mobile/src/libs/hooks/use-track-player-controls/use-track-player-controls.hook.ts

```diff
@@ -0,0 +1,68 @@
+import TrackPlayer from 'react-native-track-player';
+
+import { useCallback, useTrackPlayerProgress } from '../hooks';
+import { TRACK_SKIP_SECONDS } from './libs/constants';
+
+type Properties = {
+  isPlaying: boolean;
+};
+
+type ReturnType = {
+  handlePlayPause: () => void;
+  handleSkipToNext: () => void;
+  handleSkipToPrevious: () => void;
+  handleSkipForward: () => void;
+  handleSkipBackward: () => void;
+  handleSeek: (value: number) => void;
+};
+
+const useTrackPlayerControls = ({ isPlaying }: Properties): ReturnType => {
+  const [timeProgress] = useTrackPlayerProgress();
+
+  const handlePlayPause = useCallback(async (): Promise<void> => {
+    await (isPlaying ? TrackPlayer.pause() : TrackPlayer.play());
+  }, [isPlaying]);
+
+  const handleSkipToNext = useCallback(async (): Promise<void> => {
+    await TrackPlayer.skipToNext();
+  }, []);
+
+  const handleSkipToPrevious = useCallback(async (): Promise<void> => {
+    await TrackPlayer.skipToPrevious();
+  }, []);
+
+  const handleSkipForward = useCallback(async (): Promise<void> => {
+    await TrackPlayer.seekTo(timeProgress + TRACK_SKIP_SECONDS);
+  }, [timeProgress]);
+
+  const handleSkipBackward = useCallback(async (): Promise<void> => {
+    await TrackPlayer.seekTo(timeProgress - TRACK_SKIP_SECONDS);
+  }, [timeProgress]);
+
+  const handleSeek = useCallback(async (value: number): Promise<void> => {
+    await TrackPlayer.seekTo(value);
+  }, []);
+
+  return {
+    handlePlayPause: (): void => {
+      void handlePlayPause();
+    },
```

Comment:
```suggestion
    handlePlayPause,
```

Reply:
If I do it like this, I get an error:  error  Promise-returning function provided to property where a void return was 
expected  @typescript-eslint/no-misused-promises